modules/c++/math/unittests/test_nChooseK.cpp

```diff
@@ -0,0 +1,55 @@
+/* =========================================================================
```

Comment:
The convention is snake_case for unit test filenames so this would be `test_n_choose_k.cpp`